pandas/tests/apply/test_frame_apply.py

```diff
@@ -1398,11 +1451,17 @@ def test_apply_empty_list_reduce():
 def test_apply_no_suffix_index():
     # GH36189
     pdf = DataFrame([[4, 9]] * 3, columns=["A", "B"])
-    result = pdf.apply(["sum", lambda x: x.sum(), lambda x: x.sum()])
-    expected = DataFrame(
-        {"A": [12, 12, 12], "B": [27, 27, 27]}, index=["sum", "<lambda>", "<lambda>"]
-    )
-
+    result = pdf.apply([np.square, lambda x: x, lambda x: x])
```

Comment:
Changed from sum to square because `lambda x: x.sum()` would fail for apply; this is now consistent with

    pdf.apply(lambda x: x.sum())

which will fail on master.